packages/next/test/unit/fixtures/02-edge-function-basepath/index.test.js

```diff
@@ -8,6 +8,12 @@ jest.setTimeout(ms('6m'));
 
 describe(`${__dirname.split(path.sep).pop()}`, () => {
   it('should normalize routes in build results output', async () => {
+    if (process.platform === 'darwin') {
```

Comment:
Should we move these to integration tests so they only run on Ubuntu? They aren't really unit tests since they are running a build.